qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -14,27 +14,55 @@
 
 from __future__ import annotations
 
+import logging
+
 from functools import lru_cache
 from typing import Union, List
 
 import numpy as np
 
 from qiskit.opflow import PauliSumOp
-from qiskit.quantum_info.operators import Pauli
+from qiskit.quantum_info.analysis.z2_symmetries import Z2Symmetries
+from qiskit.quantum_info.operators import Pauli, PauliList, SparsePauliOp
 
 from qiskit_nature.second_q.operators import FermionicOp
 from .fermionic_mapper import FermionicMapper
 
+logger = logging.getLogger(__name__)
+
 
 class ParityMapper(FermionicMapper):  # pylint: disable=missing-class-docstring
-    def __init__(self):
+    def __init__(self, num_particles: tuple[int, int] | None = None):
         """The Parity fermion-to-qubit mapping.
 
-        When using this mapper ``two_qubit_reduction`` can optionally be used for the qubit
-        operator that is created, see converter class
-        :class:`~qiskit_nature.second_q.mappers.QubitConverter`.
+        When using this mapper, :attr:`num_particles` can optionally be used to apply an additional step
+        of reduction after the mapping to pauli operators. The two-qubit reduction eliminates the central
+        and last qubit in a list of Pauli that has diagonal operators (Z,I) at those positions.
+
+        Chemistry specific method:
+        It can be used to taper two qubits when the spin orbitals are ordered in two spin sectors,
+        (block spin order) according to the number of particles in the system.
```

Comment:
Can we rephrase this a little bit? I am thinking of the following aspects:
- since this is now in Nature, we do not need to highlight that this is a chemistry specific method. It should also be clear from context since this is a `FermionicMapper`
- the block ordering assumption is a bit duplicated here. Its pointed out at the end, plus it is stated explicitly that the middle and last qubit are removed. Maybe this can be combined into one statement